ui/src/app/components/workflows/workflows.component.spec.ts

```diff
@@ -65,4 +72,106 @@ describe('WorkflowsComponent', () => {
       });
     }),
   );
+
+  describe('isWorkflowHighlighted', () => {
+    it(
+      'should return true when url contains input id',
+      waitForAsync(() => {
+        const idMatching = 555;
+        router.navigate(['workflows/show', idMatching]).then(() => {
+          expect(underTest.isWorkflowHighlighted(idMatching)).toBeTruthy();
+        });
+      }),
+    );
+
+    it(
+      'should return false when url does not contain input id',
+      waitForAsync(() => {
+        const idNonMatching = 5;
+        router.navigate(['workflows/show', 555]).then(() => {
+          expect(underTest.isWorkflowHighlighted(idNonMatching)).toBeFalse();
+        });
+      }),
+    );
+  });
+
+  it(
+    'openCloseProject() should toggle a project',
```

Comment:
nitpick: `toggleProject` instead of `openCloseProject` might be a bit easier to understand

Reply:
Yes, you are right. Fixed